### ISSUE TITLE
Speed scaler service uses percent, unlike ROS1

### ISSUE THREAD

**Issue by AndyZe** [PickNikRobotics/Universal_Robots_ROS2_Driver#77]
In ROS1, the speed scaling slider used a fraction (0.01-1.0, I believe)

**Comment by gavanderhoorn** [comment]
See the [documentation](http://docs.ros.org/en/melodic/api/ur_msgs/html/srv/SetSpeedSliderFraction.html):

```yaml
# Set the speed slider on the teach pendant to the specified value.
#
# Values for 'speed_slider_fraction' must be from [0; 1.0]; values outside this
# valid range will result in an error being returned to the caller.
```

**Comment by MGBla** [comment]
Reverted back to range of 0-1.0 and added a warning if range is exceeded. #88

**Comment by MGBla** [comment]
@gavanderhoorn I think it would make sense to add an message field to the service response giving the user some info about why the service call failed or are there reasons against it?

**Comment by gavanderhoorn** [comment]
I would suggest to discuss this with @fmauch, as he was the one who designed that service definition.

**Comment by fmauch** [comment]
Absolutely not against it.

I had in mind, that the only thing going wrong would be the user's input range, but of course another possibility is that the [corresponding library function](https://github.com/UniversalRobots/Universal_Robots_Client_Library/blob/master/src/rtde/rtde_writer.cpp#L65) returns `false`.

I see the benefit of poking the user that the entered value is out-of-range.

**Comment by destogl** [comment]
solved in #88
